modules/es-extensions/publisher/extensions/app/greg-diff-api/app.js

```diff
@@ -0,0 +1,45 @@
+/*
+ * Copyright (c) 2014, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ *
+ * WSO2 Inc. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+app.server = function(ctx){
+    return {
+        endpoints:{
+            apis:[
+                {
+                    url:'governance-artifacts',
+                    path:'governance-artifacts.jag',
+                    secured:true
+                }
+            ]
+        }
+    };
+};
+
+
+app.apiHandlers = function(ctx) {
+    return {
+        onApiLoad: function() {
+            if ((ctx.isAnonContext) && (ctx.endpoint.secured)) {
+                print('{ error:"Authentication error" }'); //TODO: Fix this to return a proper status code
```

Comment:
Thanks @thushara35 , I have removed it in [this commit](https://github.com/wso2/product-greg/pull/854/commits/c82d1167588d480afbfa6def40616714b1fc3eac)